lighthouse-core/scripts/build-report-for-autodeployment.js

```diff
@@ -22,8 +26,26 @@ lhr.categories['lighthouse-plugin-someplugin'] = {
   auditRefs: [],
 };
 
-console.log('🕒 Generating report for sample_v2.json...');
-const html = ReportGenerator.generateReport(lhr, 'html');
-const filename = path.join(__dirname, '../../dist/index.html');
-fs.writeFileSync(filename, html, {encoding: 'utf-8'});
-console.log('✅', filename, 'written.');
+(async function() {
+  const filenameToLhr = {
+    'english': lhr,
+    'espanol': swapLocale(lhr, 'es').lhr,
+    'arabic': swapLocale(lhr, 'ar').lhr,
```

Comment:
can we do `en-xa-local` once #9192 lands? 😃

Reply:
yup. definitely will.